client/src/pages/Contact/Contact.tsx

```diff
@@ -42,59 +44,63 @@ export default function ContactForm() {
   }
 
   return (
```

Comment:
i would add a background form like the singup has the grey background cause it's very hard to see what to fill. (Think about color blind people).
This is also for the about page
and this is for sure NICE TO HAVE